support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -467,6 +469,20 @@ export function CheckoutComponent({
 	const returnToLandingPage = `/${geoId}${productLanding(productKey)}`;
 	const isAdLite = productKey === 'GuardianAdLite';
 
+	const contributionType =
```

Comment:
Maybe we should reconsider this variable name? Referring to the billing period as `contributionType` feels like a hangover of when we were just selling different types of contribution?  It's not really a "contribution type" at Tiers 2 and 3 anymore, so doesn't seem appropriate terminology? It feels like `paymentFrequency` or just `billingPeriod` seems more appropriate? 

Also is this mapping of 'Monthly' to 'MONTHLY', 'Annual' to 'ANNUAL', 'OneTime' to 'ONE_OFF' something we do a lot? If it's something we often do maybe we should centralise into a helper?

Looking at it maybe this is a bigger refactor throughout the code to revisit the `ContributionType` type, and something to look at in the future? what do you think @paul-daniel-dempsey?